templates/base.html

```diff
@@ -1,5 +1,5 @@
 {% extends "base_root.html" %}
-{% load i18n static cms_tags menu_tags sekizai_tags %}
+{% load i18n staticfiles cms_tags menu_tags sekizai_tags %}
```

Comment:
There were additional static vs staticfiles loads in the explorer (like 5 or 6). We should do it for all templates